examples/auth.py

```diff
@@ -0,0 +1,70 @@
+import os
```

Comment:
Maybe add a comment at the top describing what the example is actually about. And name the file something more descriptive (maybe webui_auth.py or something)